code/headers/manual_control.hpp

```diff
@@ -0,0 +1,161 @@
+#pragma once
+
+#include <base_module.hpp>
+#include <controller_interface.hpp>
+
+namespace r2d2::manual_control {
+    class module_c : public base_module_c {
+    private:
+        r2d2::manual_control::controller_interface_c &controller;
+
+        // lists to save previous states of buttons, sliders and joysticks
+        std::array<unsigned char, 2> slider_list;
```

Comment:
why an unsigned char is used here instead of an uint8_t do you store characters or just numbers?